examples/demo/src/types/FacetHit.ts

```diff
@@ -1,14 +1,3 @@
-import { Hit } from '@algolia/client-search';
+import { TrendingFacetHit } from '@algolia/recommend';
 
-export type FacetRecord = {
-  facetName: string;
-  facetValue: string;
-};
-
-type WithInsights<THit> = THit & {
-  __position: string;
-  __indexName: string;
-  __queryID: string;
-};
-
-export type FacetHit = WithInsights<Hit<FacetRecord>>;
+export type FacetHit = TrendingFacetHit<string>;
```

Comment:
I don't think `TrendingFacetHit` is generic anymore?

Reply:
Indeed, i'm currently making changes to use `v4.22.0` should have the PR ready soon.